### ISSUE TITLE
RuntimeError: CUDA out of memory. Tried to allocate 50.00 MiB (GPU 0; 7.79 GiB total capacity; 5.98 GiB already allocated; 22.88 MiB free; 6.40 GiB reserved in total by PyTorch)

### ISSUE THREAD

**Issue by zhengjie6** [dbolya/yolact#648]
When I run the following:
`python train.py --config=yolact_base_config --batch_size=8`

I keep running into this error
`RuntimeError: CUDA out of memory. Tried to allocate 50.00 MiB (GPU 0; 7.79 GiB total capacity; 5.98 GiB already allocated; 22.88 MiB free; 6.40 GiB reserved in total by PyTorch)`

I have tried decreasing the maximum size of the pictures in `max_size` under config.py but the CUDA error continue to persist.

Thanks in advance for any help

**Comment by zhengjie6** [comment]
Update: I have tried decreasing and changing the batch size as well and it does not seem to be able to solve the problem.

**Comment by kidpaul94** [comment]
Have you seen this issue https://github.com/dbolya/yolact/issues/158?

**Comment by zhengjie6** [comment]
Yes. I have tried to decrease the `max_size` variable in the config file, but it still does not work. I am using the default coco dataset 2017 for the training.

**Comment by kidpaul94** [comment]
This issue depends on quite a lot of things (e.g., your GPU, other programs running under your GPU, and etc) https://github.com/dbolya/yolact/issues/296. If you get frustrated by the unidentifiable bug, I recommend you to move your stuff to Google Colab. It will provide you a free 11 ~ 13 GB GPU. The example for using YOLACT with Colab is: https://www.immersivelimit.com/tutorials/yolact-with-google-colab
